dockerbuild-ts-dbc/Pulumi.yaml

```diff
@@ -0,0 +1,10 @@
+# Configure project names and descriptions with values obtained from `pulumi new`
+name: ${PROJECT}
+description: ${DESCRIPTION}
+runtime: nodejs
+
+template:
+  description: An example that builds a Dockerfile in Docker Build Cloud.
+  config:
+    builder:
```

Comment:
Can you confirm that this forces this value to be required? (I don't see a `required` option in [the docs](https://www.pulumi.com/docs/concepts/projects/project-file/#config-options).)